base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -72,19 +72,25 @@ pub struct Validators<B: BlockchainBackend> {
     block: Arc<Validator<Block, B>>,
     orphan: Arc<Validator<Block, B>>,
     horizon_state_header: Arc<Validator<BlockHeader, B>>,
+    chain_gb: Arc<Validator<BlockHeader, B>>,
+    chain_tip: Arc<Validator<BlockHeader, B>>,
```

Comment:
The docstring is inconsistent with the struct now. Perhaps expand it to cover all 5 validator types

Reply:
Expanded the docstring to include all 5 validators.